src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -520,7 +520,7 @@ private int ExpireCollection(CookieCollection cc)
             }
         }
 
-        internal void Add(CookieCollection cookies)
+        public void Add(CookieCollection cookies)
```

Comment:
For all of the newly exposed methods, please make sure there are tests. If there already are some in the unit tests, great.  Otherwise, please add some in the functional tests.